File: Data Structures & Algorithms/products-of-array-discluding-self/submission-2.cpp
class Solution {
public:
    vector<int> productExceptSelf(vector<int>& nums) {
        vector<int>res;
        int element = 1;
        for (int i = 0; i < nums.size(); i++)
        {
            element = 1;
            for (int j = 0; j < nums.size();j++)
            {
                if (j == i)
                    continue;
                element *= nums[j];
            }
            res.push_back(element);
        }
        return (res);
    }
};

File: Data Structures & Algorithms/top-k-elements-in-list/submission-2.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        #store the count
        count = {}
        for num in nums:
            count[num] = 1 + count.get(num, 0)
        #store in by frequency
        storage = [[] for i in range(len(nums) + 1)]
        for index, val in count.items():
            storage[val].append(index)
        #result
        res = []
        for i in range(len(nums), 0 , -1):
            for num in storage[i]:
                res.append(num)
                if (len(res) == k):
                    return res
        return res